teuthology/task/ceph_ansible.py

```diff
@@ -308,7 +308,22 @@ def get_host_vars(self, remote):
             roles = self.ctx.cluster.remotes[remote]
             dev_needed = len([role for role in roles
                               if role.startswith('osd')])
-            host_vars['devices'] = get_scratch_devices(remote)[0:dev_needed]
+            if teuth_config.get('ceph_ansible') and \
+                    self.ctx.machine_type in teuth_config['ceph_ansible']['has_lvm_scratch_disks']:
+                devices = get_file(remote, "/scratch_devs").split()
+                vols = []
+
+                for dev in devices:
+                   if 'vg_nvme' in dev:
+                       splitpath = dev.split('/')
```

Comment:
This is py3 incompatible, since get_file returns a binary data

Reply:
@kshtsk I can adda follow-up PR with the following change if that looks right to you?

`devices = get_file(remote, "/scratch_devs").decode().split()`